test/functional/api/v2/media_controller_test.rb

```diff
@@ -26,7 +26,7 @@ class Api::V2::MediaControllerTest < ActionController::TestCase
     assert_difference('Medium.count', +1) do
       post :create, { :medium => new_medium }
     end
-    assert_response :created
+    assert_response :success
```

Comment:
Can't use response.new_record? because the record already entered the db at process_success stage.